great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py

```diff
@@ -38,9 +38,17 @@
     )
     from great_expectations.render.renderer_configuration import AddParamArgs
 
+EXPECTATION_SHORT_DESCRIPTION = (
+    "Expect the set of distinct column values to be contained by a given set."
+)
+COLUMN_ARG = "The column name."
```

Comment:
You probably should move this up into `expectation.py` and import it so you don't need to keep redefining it and can use it to add the `column` field `description`s up there.